sssom/writers.py

```diff
@@ -300,13 +301,75 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
       - ConcpetMap::SSSOM mapping spreadsheet: https://docs.google.com/spreadsheets/d/1J19foBAYO8PCHwOfksaIGjNu-q5ILUKFh2HpOCgYle0/edit#gid=1389897118
 
     TODOs
-    todo: when/how to conform to R5 instead of R4?: https://build.fhir.org/conceptmap.html
-    TODO: Add additional fields from both specs
+     todo: when/how to conform to R5 instead of R4?: https://build.fhir.org/conceptmap.html
+     TODO: Add additional fields from both specs
      - ConceptMap spec fields: https://www.hl7.org/fhir/r4/conceptmap.html
       - Joe: Can also utilize: /Users/joeflack4/projects/hapi-fhir-jpaserver-starter/_archive/issues/sssom/example_json/minimal.json
-     - SSSOM more fields:
-     - prefix_map
-     - SSSOM spec fields: https://mapping-commons.github.io/sssom/Mapping/
+     - SSSOM https://mapping-commons.github.io/sssom/Mapping/
```

Comment:
All fields that need to be mapped between SSSOM and FHIR. I think we should merge this PR before full completion. I think it is going to not be any time soon before these are mapped completely.